Framework/API/src/MultipleFileProperty.cpp

```diff
@@ -33,6 +33,21 @@ Mantid::Kernel::Logger g_log("MultipleFileProperty");
 bool doesNotContainWildCard(const std::string &ext) {
   return std::string::npos == ext.find('*');
 }
+
+static const std::string SUCCESS("");
+
+// Regular expressions for any adjacent + or , operators
+static const std::string REGEX_INVALID = "\\+\\+|,,|\\+,|,\\+";
+// Regular expressions that represent the allowed instances of + or ,
+// operators.
+static const std::string REGEX_NUM_COMMA_ALPHA("(?<=\\d)\\s*,\\s*(?=\\D)");
+static const std::string REGEX_ALPHA_COMMA_ALPHA("(?<=\\D)\\s*,\\s*(?=\\D)");
+static const std::string REGEX_NUM_PLUS_ALPHA("(?<=\\d)\\s*\\+\\s*(?=\\D)");
+static const std::string REGEX_ALPHA_PLUS_ALPHA("(?<=\\D)\\s*\\+\\s*(?=\\D)");
+static const std::string REGEX_COMMA_OPERATORS =
+    REGEX_NUM_COMMA_ALPHA + "|" + REGEX_ALPHA_COMMA_ALPHA;
+static const std::string REGEX_PLUS_OPERATORS =
+    REGEX_NUM_PLUS_ALPHA + "|" + REGEX_ALPHA_PLUS_ALPHA;
```

Comment:
Would it be better to make the `boost::regex` objects static const? Or are any of these strings used in other contexts?